.azurepipelines/Platform-Build-GCC5.yml

```diff
@@ -14,7 +14,7 @@ resources:
       ref: main
   containers:
     - container: linux-gcc
-      image: ghcr.io/tianocore/containers/fedora-37-test:54e5bd1
+      image: ghcr.io/microsoft/mu_devops/ubuntu-22-test:latest

```

Comment:
Can you please update the container image used in [Release.yml](https://github.com/microsoft/mu_tiano_platforms/blob/main/.azurepipelines/Release.yml) as well?